### ISSUE TITLE
Adding own dictionary not working

### ISSUE THREAD

**Issue by jobtijhuis** [Jason-Rev/vscode-spell-checker#231]
I added a [chemistry dictionary](http://www.chemistry-blog.com/2008/12/17/chemistry-dictionary-for-word-processors-version-20/) it is just a .dic file with a word on each line. Added it to the settings but the words in the dictionary seem to be marked as wrong. There are no errors reported anywhere yet the dictionary does not seem to work.
``` 
    "cSpell.dictionaryDefinitions": [
        { "name": "chemistry", "path": "C:/Users/user/Documents/Dictionaries/chemistry-en.dic"}
    ],
    "cSpell.dictionaries": [
        "chemistry"
    ],
```

**Comment by Jason3S** [comment]
The error reporting is not the greatest.

This feature does work. The question is why it is not working in this situation.

Possible issues:
1. Settings didn't get read - restart VS Code, with some workspace settings, a restart is necessary to take effect.
2. the file isn't found - I don't use Windows, so it is possible that you need to replace the slashes.

To find out if the file failed to load:
```
"cSpell.logLevel": "Debug"
```

Choose the `OUTPUT` tab and select `Code Spell Checker`
![image](https://user-images.githubusercontent.com/3740137/41498423-ab0eabb8-716d-11e8-88e4-b7367ecd45ff.png)

There will be lots of messages, but if it doesn't load, you will see something like this:
![image](https://user-images.githubusercontent.com/3740137/41498433-e7a8d51c-716d-11e8-86e3-458c234fb141.png)

**Comment by jobtijhuis** [comment]
Thank you for answering albeit my bad reporting, I had tried the debug loglevel but since I am not the most experienced VSCode user I did not know where the logs ended up. 
Maybe it is good to display the incorrect loading of the dictionary in the configuration info somewhere.
